specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimtenant.json

```diff
@@ -34,6 +34,59 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/tenant": {
+      "get": {
+        "tags": [
+          "TenantAccess"
```

Comment:
>TenantAccess [](start = 11, length = 12)

Is the API for tenant or tenantAccess?

Reply:
For tenantAccees. We don't have "tenantAccesss" collections, so we created "/tenant/access" indicating this entity belongs to entire tenant.